configCivcraft.yml

```diff
@@ -695,9 +697,9 @@ recipes:
   Craft_Anvils_Advanced:
     type: PRODUCTION
     name: Craft Anvils
-    production_time: 5s
+    production_time: 10s
     input:
-      iron blocks:
+      iron block:
```

Comment:
That's not an identifier, it's a display name, there are hundreds of them with spaces.